<?xml version="1.0" ?>
<sdf version="1.4">
  <model name="pringles_paprika">
    <link name="pringles_paprika_link">
      <inertial>
        <pose>0 0 0 0 0 0</pose>
        <mass>0.08</mass>
      </inertial>

      <visual name="pringles_paprika_visual">
        <pose>0 0 0 0 0 0</pose>
        <material>
          <script>
            <uri>model://pringles_paprika/materials/scripts</uri>
            <uri>model://pringles_paprika/materials/textures</uri>
            <name>pringles_paprika</name>
          </script>
        </material>
        <geometry>
          <mesh>
            <uri>model://pringles_paprika/meshes/pringles_paprika.dae</uri>
          </mesh>
        </geometry>
      </visual>


      <collision name="pringles_paprika_collision">
        <pose>0 0 0 0 0 0</pose>
        <geometry>
          <mesh>
            <uri>model://pringles_paprika/meshes/pringles_paprika.dae</uri>
          </mesh>
        </geometry>
        <surface>
          <friction>
            <ode>
              <mu>0.8</mu>
              <mu2>0.8</mu2>
              <fdir1>0.0 0.0 0.0</fdir1>
              <slip1>0.01</slip1>
              <slip2>0.01</slip2>
            </ode>
          </friction>
        </surface>
      </collision>

    </link>
  </model>
</sdf>